src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
To avoid cache issue on the start project, you can remove all of part related to serviceWorker